Dockerfile

```
@@ -0,0 +1,17 @@
+FROM python:3.9.18-alpine3.18 as base
+
+RUN pip install "poetry==1.3.2"
+
+COPY ./todo_app ./todo_app
+COPY ./poetry.lock .
+COPY pyproject.toml .
+
+RUN poetry install
+
+FROM base as production
+ENTRYPOINT poetry run gunicorn --bind 0.0.0.0 "todo_app.app:create_app()"
+EXPOSE 8000 
+
+FROM base as development
+ENTRYPOINT poetry run flask run
```

If we want the container accessible externally, we'll need to similarly expose it on 0.0.0.0, and if you want it to use the same port as you've exposed/specified in the instructions we'll need to specify that too:
```suggestion
ENTRYPOINT poetry run flask run --host 0.0.0.0 --port 8000
```